swap/src/cli/tracing.rs

```diff
@@ -88,6 +96,19 @@ fn debug_terminal_printer<S>() -> StdErrPrinter<StdErrLayer<S, ChronoLocal>> {
     }
 }
 
+fn debug_json_terminal_printer<S>() -> StdErrPrinter<StdErrJsonLayer<S, ChronoLocal>> {
+    let is_terminal = atty::is(atty::Stream::Stderr);
+    StdErrPrinter {
+        inner: fmt::layer()
+            .with_ansi(is_terminal)
+            .with_target(false)
```

Comment:
The `info_json_terminal_printer` includes `.with_level(false)` - here you don't. Is that on purpose?

Reply:
yes on purpose. keeping it consistent with the non json printer.